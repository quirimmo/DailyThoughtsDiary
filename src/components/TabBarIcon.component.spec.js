import React from 'react';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import renderer from 'react-test-renderer';
import TabBarIcon from './TabBarIcon.component';

let component;

describe('TabBarIcon', () => {
	beforeEach(() => {
		jest.useFakeTimers();
		NavigationTestUtils.resetInternalState();
		component = renderer.create(<TabBarIcon name="ios-add-circle" focused={false} />);
	});
	afterEach(() => {
		jest.clearAllMocks();
	});

	it('should be defined', () => {
		expect(TabBarIcon).toBeDefined();
	});

	it('should render the component', async () => {
		expect(component.toJSON()).toMatchSnapshot();
	});
});
