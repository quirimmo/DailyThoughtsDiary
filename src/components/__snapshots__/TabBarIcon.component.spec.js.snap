// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarIcon should render the component 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;
